content/modules/redisgears/jvm/install.md

```diff
@@ -20,20 +20,37 @@ Before you can use RedisGears with the JVM, you need to install the RedisGears m
 
 ## Install RedisGears and the JVM plugin
 
-If your cluster uses Redis Enterprise v6.0.12 or later, you only need to download the RedisGears package. The JVM plugin installs automatically when you make a `POST` request to the `/v2/modules` REST API endpoint.
+If your cluster uses Redis Enterprise v6.0.12 or later and has internet access, you only need to download the RedisGears package. It automatically fetches dependencies like the JVM plugin during online installation.
 
-After the install is complete, RedisGears will appear in the list of available modules on the **settings** and **create database** pages of the Redis Enterprise admin console.
+Offline installation requires you to manually upload dependencies to the master node.
+
+To install RedisGears and the JVM plugin:
 
 1. Download the RedisGears package from the [download center](https://redis.com/redis-enterprise-software/download-center/modules/).
 
-1. Upload the package to a node in the cluster.
+    {{<note>}}
+For offline installation, you also need to download the packages for RedisGears dependencies.
```

Comment:
I do not know how it looks on the download page, maybe we should also state the files named (if its clear on the download page then its not needed)...